app/services/api/v3/ecf/participants_query.rb

```diff
@@ -57,7 +57,7 @@ def filter
         end
 
         def with_cohorts
-          return Cohort.where(start_year: filter[:cohort]) if filter[:cohort].present?
+          return Cohort.where(start_year: filter[:cohort].split(",")) if filter[:cohort].present?
```

Comment:
could you add a test for this please 😃 ?

Reply:
Just realised there are no tests for this class generally could we add one please?

Reply:
Hi @cwrw tks for your review. Addressed in https://github.com/DFE-Digital/early-careers-framework/pull/2687/commits/8faf4f1e53a01d9b0f5561e81bae9fd2b38cc091